introspection.go

```diff
@@ -318,28 +322,39 @@ func (c *Connection) IntrospectState(opts *IntrospectionOptions) ConnectionRunti
 
 // IntrospectState returns the runtime state for this relayer.
 func (r *Relayer) IntrospectState(opts *IntrospectionOptions) RelayerRuntimeState {
-	r.RLock()
-	defer r.RUnlock()
+	var count int
+	if opts.IncludeTombstones {
+		count = r.inbound.CountAll() + r.outbound.CountAll()
```

Comment:
prefer if we just maintain two separate counters, Count (only active) and Tombstoned. The "IncludeTombstones" should only affect whether we display the tombstoned relays as part of the items list